### ISSUE TITLE
Do the render methods need to add all that meta?

### ISSUE THREAD

**Issue by colonelfazackerley** [andybalaam/rabbit-escape#244]
Could these go?
```
:num_saved=0
:num_killed=0
:num_waiting=0
:paused=false
```

This roundtrips, and so things get added:
```
./runrabbit text -e ../foo.rel
```
I quite like this for some things (desc, author), as it adds placeholders to remind me. Those 4 bits I mentioned seem extraneous (and also make the tests verbose): are they necessary?

**Comment by andybalaam** [comment]
Hmm. This runtime state is necessary for the state saving we do on Android (e.g. When you rotate the phone it saves all state and restarts the app). However I do agree it is bad here and should probably be optional. Saving a level description is different from saving the current state of a playing level.

**Comment by colonelfazackerley** [comment]
OK, I will tweak so the render methods have a quiet mode that prints less, and use that for the CLI bits.

**Comment by andybalaam** [comment]
Cool

**Comment by andybalaam** [comment]
Fixed by #249
